src/serverSetup.ts

```diff
@@ -201,6 +201,7 @@ function parseServerInstallOutput(str: string, scriptId: string): { [k: string]:
 function generateBashInstallScript({ id, quality, version, commit, release, extensionIds, envVariables, useSocketPath, serverApplicationName, serverDataFolderName, serverDownloadUrlTemplate }: ServerInstallOptions) {
     const extensions = extensionIds.map(id => '--install-extension ' + id).join(' ');
     return `
+#!/bin/sh
```

Comment:
```suggestion
#!/usr/bin/env sh
```

For wider compatibility it’s best to not assume /bin/sh